integration-tests/src/test/resources/OperatorIT.properties

```diff
@@ -5,5 +5,5 @@ baseDir=/scratch
 #wls admin user
 username=weblogic
 password=welcome1
-maxIterationsPod=25
```

Comment:
update copyright notice

Reply:
reverted to original, don't need to increase to 50